### DISCUSSION TITLE
Language filter on API calls returns 500

### DISCUSSION THREAD

**Discussion by AllAstronauts** [trakt/trakt-api#442] [Q&A] [answered]
Some routine get new shows daily tasks began failing and it took a while to figure out why as the Client ID is valid and we do not come anywhere near any limits (we're talking a handful of GETS daily)

Long story short, the filter language=en (and different and/or more language codes) break routine GETS, for these APIs at least:

https://api.trakt.tv/calendars/all/shows/new/ 
https://api.trakt.tv/calendars/all/shows/premieres/

**EXAMPLES:**

https://api.trakt.tv/calendars/all/shows/new/2024-02-01/90
working

https://api.trakt.tv/calendars/all/shows/new/2024-02-01/90?countries=us
working

https://api.trakt.tv/calendars/all/shows/new/2024-02-01/90?countries=us&languages=en
**500 Response**

https://api.trakt.tv/calendars/all/shows/new/2024-02-01/90?languages=en
**500 Response**

Verified in production code AND via the https://trakt.docs.apiary.io/ calls you can make for live testing there.

This started... uhh... last 24 hours maybe? Could not be much longer than that if.

Apologies if this goes deeper or has been mentioned in some other random place I'm not checking. Have not verified any other filters; I'm just yanking the language filter on these two API calls and moving on with my day.

**Comment by rectifyer** [comment, marked as answer]
We made an internal change to use the "spoken languages" from TMDB which introduced this bug. I'll have a fix out in a few hours (the 500 won't happen anymore) and the data will populate over the next few days and improve the accuracy.
